### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'name'

### ISSUE THREAD

**Issue by allen-munsch** [aiven/aiven-client#162]
Version:

aiven_client-2.5.0_10_g923391b-py3-none-any.whl

Attempted command:

```
avn --auth-token "###" service create --project xxx-review-apps --service-type pg --plan startup-4 --cloud aws-us-east-1 -c pg_version=11 xxx-pr-2978 
```

Error:

```python

Traceback (most recent call last):
  File "/app/.heroku/python/bin/avn", line 8, in <module>
    sys.exit(main())
  File "/app/.heroku/python/lib/python3.6/site-packages/aiven/client/__main__.py", line 5, in main
    AivenCLI().main()
  File "/app/.heroku/python/lib/python3.6/site-packages/aiven/client/argx.py", line 298, in main
    sys.exit(self.run())
  File "/app/.heroku/python/lib/python3.6/site-packages/aiven/client/argx.py", line 270, in run
    return self.run_actual(args)
  File "/app/.heroku/python/lib/python3.6/site-packages/aiven/client/argx.py", line 292, in run_actual
    return func()  # pylint: disable=not-callable
  File "/app/.heroku/python/lib/python3.6/site-packages/aiven/client/cli.py", line 2730, in service__create
    service=self.args.name,

AttributeError: 'Namespace' object has no attribute 'name'
```

Any suggestions?

In the meanwhile we've rolled back to 2.3.5

**Comment by heikju** [comment]
Hi,

Thanks for the report. I believe this one was fixed here https://github.com/aiven/aiven-client/commit/3d2b5771#diff-951a771bc91b06d5a21a4438d81d9fe76f041221462253fe9c34c0e710960235R2860 , and we'll include the fix in the next release following 2.5.0.

**Comment by mts-aiven** [comment]
Fixed in 2.6.0.
